make-money/ad-overview.md

```diff
@@ -0,0 +1,22 @@
+# Ad Units
+
+## Interstitials
+
+
+Interstitials are full screen ads that are displayed in your application. Tap for Tap's interstials are MRAID and VAST compliant, with smart pre-caching for no lag time. The interstital sizes are 640x960 (Portrait) and 960x640 (landscape).
+
+### Moments Based Interstitials
+
+It is our goal to provide the most contextually relevant ads that fit in seamlessly with your application. This increases monetization performance,  while enhancing the user experience. Moments Based Monetization ensures that the right user, sees the right ad, at the right time.  These are the moments we monetize in your application:
```

Comment:
The sentence that starts "Moments Based Monetization ensures..." should not have any commas in it.